doc/classes/ItemList.xml

```diff
@@ -263,10 +268,16 @@
 			</argument>
 			<description>
 				Sets the foreground color of the item specified by [code]idx[/code] index to the specified [Color].
-				[codeblock]
-				var some_string = "Some text"
-				some_string.set_item_custom_fg_color(0,Color(1, 0, 0, 1) # This will set the foreground color of the first item of the control to red.
-				[/codeblock]
+				[codeblocks]
+				[gdscript]
+				var item_list = ItemList.new()
+				item_list.set_item_custom_fg_color(0,Color(1, 0, 0, 1) # This will set the foreground color of the first item of the control to red.
+				[/gdscript]
+				[csharp]
+				var itemList = new ItemList();
+				itemList.SetItemCustomFgColor(0, new Color(1, 0, 0, 1)); // This will set the foreground color of the first item of the control to red.
```

Comment:
I think we should just use the color constant for red. There's also a missing space in the GDScript version.

Same thing for `set_item_custom_bg_color`.

```suggestion
				item_list.set_item_custom_fg_color(0, Color.red) # This will set the foreground color of the first item of the control to red.
				[/gdscript]
				[csharp]
				var itemList = new ItemList();
				itemList.SetItemCustomFgColor(0, Colors.Red); // This will set the foreground color of the first item of the control to red.
```